client/app/scripts/superdesk/superdesk.js

```diff
@@ -97,8 +97,13 @@ define([
         }]);
 
     // make sure there is templates module defined
-    angular.module('templates', []);
-    modules.push('templates');
+    try {
```

Comment:
I'm just wondering if it's obvious. maybe this logic should be in main.js and not superdesk.js ?
why should it care about templates?